nntrainer/src/conv2d_layer.cpp

```diff
@@ -79,9 +79,9 @@ void Conv2DLayer::read(std::ifstream &file) { Layer::read(file); }
 
 void Conv2DLayer::save(std::ofstream &file) { Layer::save(file); }
 
-sharedTensor Conv2DLayer::forwarding(sharedTensor in) {
+sharedConstTensor Conv2DLayer::forwarding(sharedConstTensor in) {
   int status = ML_ERROR_NONE;
-  input = *in;
+  input = in->clone();
```

Comment:
is this needed?
normalization, standardization and zero_pad work out of place.

Reply:
I guess it is not needed. Fixed.